packages/two_dimensional_scrollables/test/table_view/table_test.dart

```diff
@@ -950,13 +1255,17 @@ void main() {
         (WidgetTester tester) async {
       // TODO(Piinks): Rewrite this to remove golden files from this repo when
       //  mock_canvas is public - https://github.com/flutter/flutter/pull/131631
-      // foreground, background, and precedence per mainAxis
+      //  * foreground, background, and precedence per mainAxis
```

Comment:
I realize I am packing a lot into this single golden file test, but I don't want to add more images to the repo when very soon I will be able to rewrite this into tests that use the mock canvas API when it reaches stable.